docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,73 @@
+* Persists the alert details flyout right panel selected tab in localStorage ({kibana-pull}179511[#179511]).
+* Bedrock Claude 3 Support ({kibana-pull}179304[#179304]).
+* Defend Advanced Policy Option to Opt-Out of New Optimization ({kibana-pull}179179[#179179]).
+* Persists the alert details flyout overview panel expandable section open/collapsed state in localStorage ({kibana-pull}178746[#178746]).
+* Limit the number of alerts considered for an entities risk score to the top 10,000 highest risk ({kibana-pull}178324[#178324]).
+* Adds alert suppression for New Terms rule type ({kibana-pull}178294[#178294]).
+* Allow users to edit setup field for custom rules ({kibana-pull}178131[#178131]).
+* Improves ES|QL investigation fields for detection rules ({kibana-pull}177746[#177746]).
+* Use expandable flyout in timeline ({kibana-pull}177087[#177087]).
+* Adds asset criticality advanced setting to serverless ({kibana-pull}176991[#176991]).
+* Adds tooltip to asset criticality accordion ({kibana-pull}176927[#176927]).
+* Adds Security AI assistant settings to the Stack management ({kibana-pull}176656[#176656]).
+* Adds Suppression to EQL Non-sequence based queries ({kibana-pull}176422[#176422]).
+* Enables `xpack.ml.nlp.enabled` for Security Serverless project ({kibana-pull}175358[#175358]).
+* Updates MITRE ATT&CK framework to `v14.1` ({kibana-pull}174120[#174120]).
+* Security AI Assistant persistent storage ({kibana-pull}173487[#173487]).
+
+[discrete]
+[[bug-fixes-8.14.0]]
+==== Bug fixes
+* Fixes a bug that prevented the **{esql}** Timeline tab from being turned off after you removed the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag from the {kib} user settings ({kibana-pull}182816[#182816]).
+* Fixes a bug that removed pinned events and comments in unsaved Timelines ({kibana-pull}178212[#178212]).
+* Fixes a bug in Timeline that prevented the **Show top _x_** action from showing accurate results ({kibana-pull}177213[#177213]).
+* Fixes a bug with the `is one of` Timeline filter that generated incorrect Query Domain Specific Language (DSL) queries ({kibana-pull}180455[#180455]).
+* Ensures the `securitySolution:enableAssetCriticality` advanced setting is enabled before the asset criticality levels to your entities are updated ({kibana-pull}181780[#181780]).
+* Corrects the color theme for the entity risk score UI to ensure it works on dark mode ({kibana-pull}181431[#181431]).
+* Fixes a bug that caused the Entity Analytics dashboard to make too many risk_score/index_status requests ({kibana-pull}179510[#179510]).
+* Escape username correctly ({kibana-pull}181709[#181709]).
+* Moves the `observer.serial_number` field to the Highlighted Fields section for alerts generated by SentinelOne and removes the {agent} status field ({kibana-pull}181038[#181038]).
+* Migrate bulk enable/diable rules to Alerting methods ({kibana-pull}180796[#180796]).
```

Comment:
@xcrzx can you provide a more descriptive release note summary for this bug fix?

Reply:
Maybe something like "When bulk enabled, rules now do not flood the task manager, improving performance."?

Reply:
@xcrzx how's this:

```suggestion
* Fixes an issue that caused {kib} Task Manager to become overloaded when rules were bulk enabled ({kibana-pull}180796[#180796]).
```

Reply:
Yup, that sounds good 👍